### PR TITLE
DMP-3096: Add group and role, media_in_perpetuity and media_accessor

### PR DESCRIPTION
**Pull request by tonymort**
https://tools.hmcts.net/jira/browse/DMP-3096

Add group and role, media_in_perpetuity and media_accessor

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java [hunk 1]
```diff
@@ -68,6 +68,16 @@ void setUp() {
         when(currentTimeHelper.currentOffsetDateTime()).thenReturn(OffsetDateTime.of(2023, 10, 27, 22, 0, 0, 0, ZoneOffset.UTC));
     }
 
+    @BeforeEach
```
Comment (mestebanez) [inline src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java:71]: Never seen this before so unsure whether its totally necessary but if it is could possibly be added as a juni5 rule.

Comment (tonymort) [reply src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java:71]: This mimics the open-in-view sessions strategy thats enabled on our spring deployments.  Means we don't have to introduce transactions into production code to make integration tests pass.  I will be good to attempt to role it out to all integration tests but I tried and wasn't easy.  For this PR just leaving it in this test. Will attempt to expand it in separate ticket.  It should probably be used going forward though.  I think "@ExtendWith" has replaced Rules in junit5.

File: src/main/resources/db/migration/common/V1_322__media_in_perpetuity.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO security_role (rol_id, role_name, display_name, display_state)
+    VALUES (13, 'MEDIA_ACCESSOR', 'Media Accessor', true);
+
```
[the remark this answers is not in the document]
Comment (tonymort) [reply src/main/resources/db/migration/common/V1_322__media_in_perpetuity.sql:3]: Should it be 14 or does it increment on requesting the next value?

[the remark this answers is not in the document]
Comment (chrisbellingham-hmcts) [reply src/main/resources/db/migration/common/V1_322__media_in_perpetuity.sql:3]: Ah, should be 14 according to the docs https://www.postgresql.org/docs/current/sql-altersequence.html:

`the specified value will be returned by the next call of nextval`

**Review by cmagowan-hmcts — APPROVED** [review]

**Pull request merged**